src/utils/server-utils.ts

```diff
@@ -68,9 +68,10 @@ function formatCapabilitiesInfo(
 
     const availableDates = dates
       .filter(date => !isEmpty(date))
+      .map(date => get(date, '_text', date).split('T')[0])
       .map(date =>
         // adding 12 hours to avoid  errors due to daylight saving
-        moment.utc(get(date, '_text', date)).set({ hour: 12 }).valueOf(),
+        moment.utc(date).set({ hour: 12 }).valueOf(),
```

Comment:
Not a big deal but you could just replace date with `get(date, '_text', date).split('T')[0])` in this line

Reply:
Done